SSO-Auth/Config/configPage.html

```diff
@@ -1,15 +1,272 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
+  <head>
     <title>SSO</title>
-</head>
-<body>
-    <div data-role="page" class="page type-interior pluginConfigurationPage esqConfigurationPage">
-        <div data-role="content">
-            <div class="content-primary">
-                <a href="https://github.com/9p4/jellyfin-plugin-sso/blob/main/README.md">Review the documentation. This plugin is configured via the API.</a>
+  </head>
+  <body>
+    <div
+      id="sso-config-page"
+      data-role="page"
+      class="page type-interior pluginConfigurationPage esqConfigurationPage"
+      data-controller="__plugin/SSO-Auth.js"
+    >
+      <div data-role="content">
+        <div class="content-primary">
+          <a
+            href="https://github.com/9p4/jellyfin-plugin-sso/blob/main/README.md"
+            >Review the documentation. This plugin is configured via the API.</a
+          >
+
+          <form id="sso-load-config" class="esqConfigurationForm">
+            <div class="selectContainer">
+              <label class="selectLabel" for="selectProvider"
+                >Choose OIDC Provider to load configuration for:</label
```

Comment:
Should not be a sentence and instead a phrase (if it's ending w a colon)

Maybe something like "OIDC Provider:"

Reply:
Resolved by https://github.com/9p4/jellyfin-plugin-sso/pull/18/commits/5112bb13afe9943aad8784e551d7277870ddeaae